### PR TITLE
Added 3 new bundle functions with basic unit tests

### PR DESCRIPTION
**Pull request by Pawes**
Created the new bundle functions as mentioned by #70
I am new to unit testing in Python so these unit tests are what I could manage by looking at the existing tests.

### CODE REVIEW

**Review by andrii-bodnar — APPROVED** [review]

File: crowdin_api/api_resources/bundles/resource.py [hunk 1]
```diff
@@ -127,6 +129,65 @@ def edit_bundle(self, projectId: int, bundleId: int, data: Iterable[BundlePatchR
             request_data=data,
         )
 
+        """
```
Comment (ayatsyny) [inline crowdin_api/api_resources/bundles/resource.py:132]: Could you move the docstring into the method?

File: crowdin_api/api_resources/bundles/resource.py [hunk 2]
```diff
@@ -127,6 +129,65 @@ def edit_bundle(self, projectId: int, bundleId: int, data: Iterable[BundlePatchR
...
+        projectId: int,
+        bundleId: int,
+        exportId: str):
+
+        return self.requester.request(
+            method="get",
+            path=f"{self.get_bundles_path(projectId=projectId, bundleId=bundleId, exportId=exportId)}/download",
+        )
+
+        """
```
Comment (ayatsyny) [inline crowdin_api/api_resources/bundles/resource.py:152]: the same

File: crowdin_api/api_resources/bundles/resource.py [hunk 3]
```diff
@@ -127,6 +129,65 @@ def edit_bundle(self, projectId: int, bundleId: int, data: Iterable[BundlePatchR
...
+        self,
+        projectId: int,
+        bundleId: int):
+
+        return self.requester.request(
+            method="post",
+            path=f"{self.get_bundles_path(projectId=projectId, bundleId=bundleId)}/exports",
+        )
+
+        """
```
Comment (ayatsyny) [inline crowdin_api/api_resources/bundles/resource.py:171]: the same

File: crowdin_api/api_resources/bundles/resource.py [hunk 4]
```diff
@@ -14,9 +14,11 @@ class BundlesResource(BaseResource):
     Link to documentation for enterprise:
     https://developer.crowdin.com/enterprise/api/v2/#tag/Bundles
     """
-
-    def get_bundles_path(self, projectId: int, bundleId: Optional[int] = None):
+    def get_bundles_path(self, projectId: int, bundleId: Optional[int] = None, exportId: Optional[str] = None):
```
Comment (ayatsyny) [inline crowdin_api/api_resources/bundles/resource.py:17]: Could you create another method path? It is a good idea to decompose paths in different methods.

**Comment by Pawes** [comment]
I hope this was what you were thinking @ayatsyny

File: crowdin_api/api_resources/bundles/resource.py [hunk 5]
```diff
@@ -14,13 +14,16 @@ class BundlesResource(BaseResource):
...
     https://developer.crowdin.com/enterprise/api/v2/#tag/Bundles
     """
-
     def get_bundles_path(self, projectId: int, bundleId: Optional[int] = None):
         if bundleId:
             return f"projects/{projectId}/bundles/{bundleId}"
 
         return f"projects/{projectId}/bundles"
 
+    def get_bundles_exports_path(self, projectId:int, bundleId:int,  exportId:str):
```
Comment (ayatsyny) [inline crowdin_api/api_resources/bundles/resource.py:23]: Could you change `exportId: Optional[str] = None`?

File: crowdin_api/api_resources/bundles/resource.py [hunk 6]
```diff
@@ -127,6 +130,68 @@ def edit_bundle(self, projectId: int, bundleId: int, data: Iterable[BundlePatchR
...
+        Link to documentation:
+        https://developer.crowdin.com/api/v2/#operation/api.projects.bundles.exports.post
+
+        Link to documentation for enterprise:
+        https://developer.crowdin.com/enterprise/api/v2/#operation/api.projects.bundles.exports.post
+        """
+
+        return self.requester.request(
+            method="post",
+            path=f"{self.get_bundles_path(projectId=projectId, bundleId=bundleId)}/exports",
```
Comment (ayatsyny) [inline crowdin_api/api_resources/bundles/resource.py:171]: We have the method get_bundles_exports_path. Could you use this method? We have the duplicate word `exports`. It isn't very good. You can see the method  get_bundles_path and create similar logic.

File: crowdin_api/api_resources/bundles/resource.py [hunk 7]
```diff
@@ -127,6 +132,68 @@ def edit_bundle(self, projectId: int, bundleId: int, data: Iterable[BundlePatchR
...
+        Link to documentation:
+        https://developer.crowdin.com/api/v2/#operation/api.projects.bundles.exports.post
+
+        Link to documentation for enterprise:
+        https://developer.crowdin.com/enterprise/api/v2/#operation/api.projects.bundles.exports.post
+        """
+
+        return self.requester.request(
+            method="post",
+            path=f"{self.get_bundles_exports_path(projectId=projectId, bundleId=bundleId)}",
```
Comment (ayatsyny) [inline crowdin_api/api_resources/bundles/resource.py:173]: You don't need to use the `f` string format if the method return string. Could you please remove it?

File: crowdin_api/api_resources/bundles/tests/test_bundles_resources.py [hunk 8]
```diff
@@ -159,6 +169,36 @@ def test_edit_bundle(self, m_request, base_absolut_url, value):
...
+        )
+
+    @mock.patch("crowdin_api.requester.APIRequester.request")
+    def test_export_bundle(self, m_request, base_absolut_url):
+        m_request.return_value = "response"
+
+        resource = self.get_resource(base_absolut_url)
+        assert resource.export_bundle(projectId=1, bundleId=1) == "response"
+        m_request.assert_called_once_with(
+            method="post", path=f"{resource.get_bundles_exports_path(projectId=1, bundleId=1)}"
```
Comment (ayatsyny) [inline crowdin_api/api_resources/bundles/tests/test_bundles_resources.py:189]: the same

File: crowdin_api/api_resources/bundles/tests/test_bundles_resources.py [hunk 9]
```diff
@@ -159,6 +169,36 @@ def test_edit_bundle(self, m_request, base_absolut_url, value):
...
+        )
+
+    @mock.patch("crowdin_api.requester.APIRequester.request")
+    def test_export_bundle(self, m_request, base_absolut_url):
+        m_request.return_value = "response"
+
+        resource = self.get_resource(base_absolut_url)
+        assert resource.export_bundle(projectId=1, bundleId=1) == "response"
+        m_request.assert_called_once_with(
+            method="post", path={resource.get_bundles_exports_path(projectId=1, bundleId=1)}
```
Comment (ayatsyny) [inline crowdin_api/api_resources/bundles/tests/test_bundles_resources.py:189]: also remove `{`and `}`

**Pull request merged**